plan/stats.go

```diff
@@ -174,7 +175,23 @@ func (p *LogicalSelection) deriveStats() (*statsInfo, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	p.stats = childProfile.scale(selectionFactor)
+	factor := selectionFactor
```

Comment:
how about s/factor/selecity/?

Reply:
Em, `selecity` is not a word, you mean `selectivity`?

Reply:
yeah.